src/main/features/ccj/routes/check-and-send.ts

```diff
@@ -40,7 +40,7 @@ function renderView (form: Form<Declaration>, req: express.Request, res: express
     claim: user.claim,
     draft: user.ccjDraft.document,
     defendant: defendant,
-    amountToBePaid: user.claim.totalAmount - (user.ccjDraft.document.paidAmount.amount || 0),
+    amountToBePaid: user.claim.totalAmountTillToday - (user.ccjDraft.document.paidAmount.amount || 0),
```

Comment:
Are you sure this should be changed as well? The change was only to update part admission pages and first contact view of the claim.